internal/jptrace/warning.go

```diff
@@ -13,7 +13,7 @@ const (
 	// store various warnings produced from transformations,
 	// such as inbound sanitizers and outbound adjusters.
 	// The value type of the attribute is a string slice.
-	WarningsAttribute = "jaeger.internal.warnings"
```

Comment:
@yurishkuro I wonder if it'd be nice to have all of these in one file. Maybe under `internal/jptrace/attributes.go`?

Reply:
Yes absolutely. I thought it was implied in the issue.